libs/ui/src/contest_tally.tsx

```diff
@@ -57,33 +57,33 @@ const ContestTable = styled.table`
   & tr.metadata {
     font-size: 0.75em;
   }
-  & tr.metadata.last {
-    border-bottom: 1px solid #808080;
+  & tr.metadata.row {
```

Comment:
This class naming confuses me because all the `tr`'s are rows, and it doesn't tell me what's different about this row.

Reply:
👍  Updated `row` to `metadatarow`.